cmd/gb/gb_test.go

```diff
@@ -589,6 +589,17 @@ func TestInfoCmd(t *testing.T) {
 	gb.grepStdout(`^GB_GOROOT="`+regexp.QuoteMeta(runtime.GOROOT())+`"$`, "missing GB_GOROOT")
 }
 
+func TestInfoWithArgs(t *testing.T) {
+	gb := T{T: t}
+	defer gb.cleanup()
+
+	gb.tempDir("src")
+	gb.cd(gb.tempdir)
+	gb.run("info", "GB_PROJECT_DIR", "GB_GOROOT")
```

Comment:
It returns a blank line for each invalid variable, for API simplicity (similar to `go env`).

Example:
```sh
$ gb env GB_PROJECT_DIR GB_UNKNOWN GB_GOROOT
/Users/gosuri/code/go

/usr/local/Cellar/go/1.5.2/libexec
```